Steepfile

```diff
@@ -11,10 +11,12 @@ target :lib do
 
   library "set", "pathname", "json", "logger", "monitor", "tsort", "uri", 'yaml', 'dbm', 'pstore', 'singleton', 'shellwords', 'fileutils', 'find', 'digest'
   signature "stdlib/strscan/0/"
-  signature "stdlib/rubygems/0/"
   signature "stdlib/optparse/0/"
   signature "stdlib/rdoc/0/"
 
+  # Please remove the following line after releasing #1107
+  signature "core/rubygems/"
```

Comment:
Steep uses released RBS, so we need to specify the `core/rubygems` directory until this PR is released to RubyGems.